app/api/models.py

```diff
@@ -0,0 +1,82 @@
+from enum import Enum
+
+from sqlalchemy import ForeignKey, Integer, String
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from app.core.constance import CATEGORY_NAME_LENGTH, STRING_LENGTH
+from app.core.db import Base
+
+
+class SpeakingSkillEnum(str, Enum):
+    begginer = 'begginer'
+    pre_intermidiate = 'pre-intermidiate'
```

Comment:
Also typo in "pre_intermidiate", should be "pre_interm**e**diate". I use *Code Spell Checker* extension to catch such small things. However it doesn't know words like "sqlalchemy" and highlights them. But after marking word as exception, it doesn't highlight it anymore